app/views/ConfirmationView.scala.html

```diff
@@ -17,34 +17,32 @@
 @import config.FrontendAppConfig
 
 @this(
-    appConfig: FrontendAppConfig,
-    main_template: MainTemplate
+        appConfig: FrontendAppConfig,
+        govukWrapper: GovukWrapper
 )
 
 @()(implicit request: Request[_], messages: Messages)
 
-@main_template(title = messages("confirmation.title")){
+@govukWrapper(title = messages("confirmation.title")){
 
-    <div class="govuk-box-highlight">
-        @components.heading("confirmation.heading", "heading-xlarge")
+
+    <div class="govuk-panel govuk-panel--confirmation">
+        @components.heading("confirmation.heading", "govuk-panel__title")
     </div>
 
-    <h2 class="heading-medium">@messages("confirmation.whatHappensNext.title")</h2>
+    <h2 class="govuk-heading-m">@messages("confirmation.whatHappensNext.title")</h2>
 
-    <p>@messages("confirmation.whatHappensNext.line1")</p>
+    <p class="govuk-body">@messages("confirmation.whatHappensNext.line1")</p>
 
     <div>
-        <p>@messages("confirmation.whatHappensNext.line2")
+        <p class="govuk-body">@messages("confirmation.whatHappensNext.line2")
             <a target="_blank" id="companies-house-search" href="@appConfig.companyHouseRegisterUrl">
```

Comment:
This <a> needs the `govuk-link` class adding I think, currently it's the wrong blue